code/acting/potential_field_sim/test.py

```
@@ -0,0 +1,63 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy.ndimage import rotate
```

Ruff (0.8.2)

`scipy.ndimage.rotate` imported but unused

Remove unused import: `scipy.ndimage.rotate`

(F401)